MappingGenerator/MappingGenerator/MappingGenerator/Features/CodeFixes/ExplicitConversionCodeFixProvider.cs

```diff
@@ -43,7 +48,12 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
             switch (statement)
             {
                 case AssignmentExpressionSyntax assignmentExpression:
-                    context.RegisterCodeFix(CodeAction.Create(title: title, createChangedDocument: c => GenerateExplicitConversion(context.Document, assignmentExpression, c), equivalenceKey: title), diagnostic);
+                    context.RegisterCodeFix(CodeAction.Create(title: title, createChangedDocument: c => GenerateExplicitConversion(context.Document, assignmentExpression, false, c), equivalenceKey: title), diagnostic);
+
+                    if (assignmentExpression.Parent is InitializerExpressionSyntax || assignmentExpression.Parent is StatementSyntax)
+                    {
+                        context.RegisterCodeFix(CodeAction.Create(title: title2, createChangedDocument: c => GenerateExplicitConversion(context.Document, assignmentExpression, true, c), equivalenceKey: title), diagnostic);
```

Comment:
@cezarypiatek after testing the provided VSIX on #146, I've noticed that the new fix suggestion doesn't appears.
I've found that you have a little error here, you need to use **title2** for "equivalenceKey", when I test the extension with title2 the fix suggestion appears and works perfectly :) 
```suggestion
                        context.RegisterCodeFix(CodeAction.Create(title: title2, createChangedDocument: c => GenerateExplicitConversion(context.Document, assignmentExpression, true, c), equivalenceKey: title2), diagnostic);
```

Reply:
Thanks, should be fixed now.